org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JavaClientConnection.java

```diff
@@ -77,7 +77,7 @@ public interface JavaLanguageClient extends LanguageClient, ExecuteCommandPropos
 	}
 
 	private final LogHandler logHandler;
-	private final JavaLanguageClient client;
+	public final JavaLanguageClient client;

```

Comment:
If you make this package-private (no modifier here), it should still work as they're in the same package.